treeherder/model/models.py

```diff
@@ -204,8 +209,12 @@ def search(cls, search_term):
         # 90 days ago
         time_limit = datetime.datetime.now() - datetime.timedelta(days=90)
 
+        # Replace MySQL's Full Text Search Operators with spaces so searching
+        # for errors that have been pasted in still works.
+        sanitised_term = re.sub(mysql_fts_operators_re, search_term, " ")
+
         # Wrap search term so it is used as a phrase in the full-text search.
-        search_term_fulltext = '"%s"' % search_term.replace("\"", "").replace("-", " ")
+        search_term_fulltext = '"%s"' % sanitised_term.replace("\"", "")
```

Comment:
I think the `.replace()` here can now be removed, since all double quotes will have already been replaced by the regex above. (I think it's also fine for the behaviour change of replacing the double quotes with spaces for parity with the other characters and simplicity)